ff/serving/coordinator/coordinator.go

```diff
@@ -197,17 +204,17 @@ func (c *Coordinator) runSQLTransformationJob(transformSource *metadata.SourceVa
 	}
 	providerResourceID := provider.ResourceID{Name: resID.Name, Variant: resID.Variant, Type: provider.Transformation}
 	transformationConfig := provider.TransformationConfig{TargetTableID: providerResourceID, Query: query}
-	c.Logger.Debug(fmt.Sprintf("%v", transformationConfig))
 	if err := offlineStore.CreateTransformation(transformationConfig); err != nil {
 		return err
 	}
-	if err := c.Metadata.SetStatus(context.Background(), resID, string(metadata.READY)); err != nil {
+	if err := c.Metadata.SetStatus(context.Background(), resID, pb.ResourceStatus{Status: pb.ResourceStatus_READY}); err != nil {
```

Comment:
Does it make more sense to do:
SetStatus(ctx, redID, pb.ResourceStatus_READY)
and have SetStatus nest the message? Feel like it would look cleaner and be easier to interact with

Reply:
yes it would, I'll change it